Backend/dev/migrations/dynamic-offer-driver-app/0230-add-vehicle-variant-in-driver-pool-config.sql

```diff
@@ -0,0 +1,21 @@
+ALTER TABLE atlas_driver_offer_bpp.driver_pool_config ADD vehicle_variant character varying(255);
+
+INSERT INTO atlas_driver_offer_bpp.driver_pool_config (
```

Comment:
You should write generic SQL queries which will work for production and master too. 
Like take all driver_pool_configs and create entry for other vehicle variants.